app/views/addresses/edit.haml

```diff
@@ -0,0 +1,19 @@
+= content_for :title, 'Adresse bearbeiten'
+
+.page-header
+  %h1= content_for :title
+
+= bootstrap_form_for @address,
+  layout: :horizontal,
+  label_col: 'col-md-2',
+  control_col: 'col-md-4' do |f|
+  = f.hidden_field :addressable_id
```

Comment:
As #73 shows we need a chain of creating all our models. 
After the edit view is fixed, I am going to change the relation from user to player.
This will enable an clear chain of model creations. 
#73 is assigned to me, I will handle this issue.